src/clojurians_log/views.clj

```diff
@@ -220,14 +220,18 @@
    [:body
     [:div.main
      (fork-me-badge)
+     [:a {:href (path-for context :clojurians-log.routes/about)}
+      "About Clojurians Slack Log"]
      [:h1 "Channels"]
-     [:ul
-      (for [{:channel/keys [name]} channels]
-        [:li
-         [:a {:href (path-for context
-                              :clojurians-log.routes/channel
-                              {:channel name})}
-          "# " name]])]]]])
+     (if (seq channels)
+       [:ul
+        (for [{:channel/keys [name]} channels]
+          [:li
+           [:a {:href (path-for context
+                                :clojurians-log.routes/channel
+                                {:channel name})}
+            "# " name]])]
+       [:p "no data loaded"])]]])
```

Comment:
Fix for #61. Question: is there a more suitable text than "no data loaded", especially for the production environment?